sidebars.js

```diff
@@ -128,6 +128,7 @@ const sidebars = {
       label: 'Platform Features',
       items: [
         'studios/platform_features/marketplace',
+        'studios/platform_features/validations',
```

Comment:
It's a bit awkward to see "Validations" split up across so many pages (C# bot-writing, JS bot-writing, and now this standalone page). At some point we may want to rename the page to be for the overall dashboard, with the validations feature as a sub-heading and forgo a lot of the re-explanation. Alternatively, restructure the sidebar layouts so that all validations content (writing + viewing) is grouped together logically